teachers_digital_platform/js/expandable-facets.js

```diff
@@ -121,12 +117,12 @@ function expandableClickHandler() {
  * @param {HTMLNode} element - The expandable target HTML DOM element.
  */
 function toggleTargetState( element ) {
-  if ( contains( element, this.classes.targetExpanded ) ) {
-    addClass( this.ui.target, this.classes.targetCollapsed );
-    removeClass( this.ui.target, this.classes.targetExpanded );
+  if ( element.classList.contains( this.classes.targetExpanded ) ) {
+    this.ui.target.classList.addClass( this.classes.targetCollapsed );
+    this.ui.target.classList.removeClass( this.classes.targetExpanded );
```

Comment:
Looks like these should just be `classList.add` and `classList.remove`.

Reply:
Thanks! Looks like that fixed it.